sdk/digitaltwins/azure-digitaltwins-core/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+    <groupId>com.azure.digitaltwins.core</groupId>
+    <artifactId>azure-digitaltwins-core</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
Versioning for libraries in the repo is expected to use the `beta` moniker, you'll want to change this to `1.0.0-beta.1`. Additionally, this will need a version comment tag as other versions in the POM file have, in this case it'll be `<!-- {x-version-update;com.azure.digitaltwins.core:azure-digitaltwins-core;current} -->`. This will also require adding a new line into this file which is used to validate and update versions automatically using a script: https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt